wolfcrypt/src/tfm.c

```diff
@@ -1739,7 +1739,7 @@ static int _fp_exptmod_nct(fp_int * G, fp_int * X, fp_int * P, fp_int * Y)
   if (M == NULL) {
      return FP_MEM;
   }
-  res = &M[1 << winsize];
+  res = &M[1ULL << winsize];
```

Comment:
Is long long the most portable?  Will long work?

Reply:
Using `1U` and `1UL`, I the following warning in a MSC 64-bit build:

    warning C4334: '<<': result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)

I believe the suffix was added in C99, so it probably isn't portable. I'm putting it back to the word32 typecast.